uaclient/tests/test_cli_attach.py

```diff
@@ -13,17 +14,45 @@
     action_attach,
     attach_parser,
     get_parser,
+    main,
     main_error_handler,
 )
 from uaclient.exceptions import (
     AlreadyAttachedError,
     LockHeldError,
+    MagicAttachInvalidParam,
+    MagicAttachTokenError,
     NonRootUserError,
     UrlError,
     UserFacingError,
 )
 from uaclient.testing.fakes import FakeFile
 
+HELP_OUTPUT = textwrap.dedent(
+    """\
+usage: pro attach <token> [flags]
+
+Attach this machine to Ubuntu Pro with a token obtained from:
+https://ubuntu.com/pro
+
+When running this command without a token, it will generate a short code
+and prompt you to attach the machine to your Ubuntu Pro account using
+a web browser.
```

Comment:
@lechsandecki what do you think about this help text for the magic-attach flow ?